src/OpenTelemetry/Internal/PeriodicExportingMetricReaderHelper.cs

```diff
@@ -39,4 +48,19 @@ internal static class PeriodicExportingMetricReaderHelper
 
         return metricReader;
     }
+
+    private static int GetValue(int? optionsValue, string envVarKey, int defaultValue)
```

Comment:
Might be worth mentioning the priority in docs? (currently I see only 2 & 3)
1. user code
2. envs
3. defaults

Reply:
I now realized that I need to describe the env vars in `README.md` files

Reply:
https://github.com/open-telemetry/opentelemetry-dotnet/pull/3424/commits/bd025a4d509ab66cb6f754e1b72f21ad9d7dc6db

Reply:
> The `Options` type setters take precedence over the environment variables

👍 seems good enough